diff --git a/node_modules/@nozbe/watermelondb/native/android/build.gradle b/node_modules/@nozbe/watermelondb/native/android/build.gradle
index 1111111..2222222 100644
--- a/node_modules/@nozbe/watermelondb/native/android/build.gradle
+++ b/node_modules/@nozbe/watermelondb/native/android/build.gradle
@@ -26,6 +26,10 @@ android {
 
     namespace "com.nozbe.watermelondb"
 
+    buildFeatures {
+        buildConfig true
+    }
+
     defaultConfig {
         minSdkVersion getExtOrIntegerDefault('minSdkVersion')
         targetSdkVersion getExtOrIntegerDefault('targetSdkVersion')
