diff --git a/node_modules/@react-native-community/geolocation/android/build.gradle b/node_modules/@react-native-community/geolocation/android/build.gradle
index 1111111..2222222 100644
--- a/node_modules/@react-native-community/geolocation/android/build.gradle
+++ b/node_modules/@react-native-community/geolocation/android/build.gradle
@@ -35,6 +35,10 @@ android {
 
   compileSdkVersion getExtOrIntegerDefault('compileSdkVersion')
 
+  buildFeatures {
+    buildConfig true
+  }
+
   defaultConfig {
     minSdkVersion getExtOrIntegerDefault('minSdkVersion')
     targetSdkVersion getExtOrIntegerDefault('targetSdkVersion')
